### PR TITLE
new

### PR DESCRIPTION
**Pull request by andrei2701**

### CODE REVIEW

File: AndreiFamishkin/01.html/task3.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<METAcharset="utf-8">
```
Comment (pluhin) [inline AndreiFamishkin/01.html/task3.html]: where is whitespace between meta and charset?
Could you please checks in all files

File: AndreiFamishkin/01.html/task1.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	<BODY> 
+<!-- Заголовок страницы -->
+		<H1>Это заголовок</H1>
+			<P><DEL>Добрый день, <EM>Уважаемые слушатели</EM></DEL>!
+			Сегодня мы с вами будем рассматривать структуру стандартного <STRONG>HTML</STRONG> файла.</P>
+			<P>Далее мы будем использовать произвольный текст для наполнения контентом. 
+			Не стоит вдумываться в его содержание.</P>
+			<P>«Си́мпсоны» (англ. The Simpsons) — самый длинный мультсериал в истории американского телевидения,
+			длящийся 25 сезонов. Первая мини-серия «Good night» была показана в «Шоу Трейси Ульман» 19 апреля 1987 года.
+<!-- начало комментария -->
```
Comment (pluhin) [inline AndreiFamishkin/01.html/task1.html]: In lines 17 and 30 you have two comments.
It should be one comment started from line 17 and will ended on line 30

File: AndreiFamishkin/01.html/task4.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        <hr>    
+        <h2>Семантические и функциональные строчные теги</h2>               
+            <p>Пожелания автора:<br>
+               <s class="redtext">Данный текст убрать из монографии</s> и заменить его на <u class="bluetext">вновь помещённый текст</u>.</p>
+        <hr>
+        <h2>Работа со строчными и блоковыми цитатами</h2>
+            <p>Цитата короткая:<br>
+               <i>И всё­таки она вертится!</i></p>
+            <p>Цитата занимающая несколько предложений:</p>
+<pre>       О сколько нам открытий чудных
```
Comment (pluhin) [inline AndreiFamishkin/01.html/task4.html]: here need blockquote  tag http://htmlbook.ru/html/blockquote

File: AndreiFamishkin/01.html/task4.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        <h2>Вот такой текст</h2>
+            <p><b><sup>Э</sup><sub>т</sub><sup>о</sup><sub>п</sub><sup>р</sup><sub>о</sub><sup>с</sup>
+                <sub>т</sub><sup>о</sup><sub>в</sub><sup>е</sup><sub>с</sub><sup>ё</sup>
+                <sub>л</sub><sup>ы</sup><sub>т</sub><sup>к</sup><sub>е</sub><sup>т</sup><sub>с</sub></b>
+            </p>
+        <hr>
+        <h2>Акроним и курсив</h2>
+            <p>ТФ:Y(x)=e<sup>-­iπx</sup>+log<sub>x</sub>4</p>
+        <hr>
+        <h2>Тег small внтури абзаца</h2>
```
Comment (pluhin) [inline AndreiFamishkin/01.html/task4.html]: And were is tag small inside p?

**Review by pluhin — CHANGES_REQUESTED** [review]
See comments + I do not see all Pythons files here

**Comment by pluhin** [comment]
Any news here?

**Comment by andrei2701** [comment]
busy on job17.01.2019, 20:59, "Siarhei Pishchyk" <notifications@github.com>:Any news here?

**Comment by andrei2701** [comment]
мда

File: AndreiFamishkin/01.html/task1.html [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+<!-- начало документа -->
+	<BODY> 
+<!-- Заголовок страницы -->
+		<H1>Это заголовок</H1>
+			<P><DEL>Добрый день, <EM>Уважаемые слушатели</EM></DEL>!
+			Сегодня мы с вами будем рассматривать структуру стандартного <STRONG>HTML</STRONG> файла.</P>
+			<P>Далее мы будем использовать произвольный текст для наполнения контентом. 
+			Не стоит вдумываться в его содержание.</P>
+			<P>«Си́мпсоны» (англ. The Simpsons) — самый длинный мультсериал в истории американского телевидения,
+			длящийся 25 сезонов. Первая мини-серия «Good night» была показана в «Шоу Трейси Ульман» 19 апреля 1987 года.
```
Comment (pluhin) [inline AndreiFamishkin/01.html/task1.html]: You did not close this paragraph

File: AndreiFamishkin/02.table/My_happy_birthday.html [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+		 table {
+        margin: auto; 
+		margin-top: 180px
+   }
+		</style>
+		</head>
+	<body>
+		<table width="200" border="1">
+			<caption>Месяц моего рождения</caption>
+			<col width="16,67"><col width="16,67"><col width="16,67"><col width="16,67"><col width="16,67"><col width="16,67">
```
Comment (pluhin) [inline AndreiFamishkin/02.table/My_happy_birthday.html:16]: dot is delimiter between fractional and integer

File: AndreiFamishkin/03.links/MaketSite/MaketSite.html [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+        <h1>Меню список</h1>
+            <ul>
+                <li><a href="./Glavnoe.html"class="bluetext">Главная</a></li>
+                <li><a href="./Catalog.html"class="bluetext">Каталог</a></li>
+                <li><a href="./Akcii.html"class="bluetext">Акции</a></li>
+                <li><a href="./O_nas.html"class="bluetext">О нас</a></li>
+            </ul>
+
+        <h1>Меню со вложенным списком</h1>
+            <ul>
```
Comment (pluhin) [inline AndreiFamishkin/03.links/MaketSite/MaketSite.html:28]: Wrong structure for nested lists

File: AndreiFamishkin/03.links/piton1.html [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+		<style>
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+		<h1>HOW-TO: Программа на Python, Часть 1</h1>
+			<ul>
+				<li><a href="./piton1.html" target="_blank"class="bluetext">Программа на Python — часть 1</a></li>
```
Comment (pluhin) [inline AndreiFamishkin/03.links/piton1.html]: do not use target blank for the links to same site

File: AndreiFamishkin/03.links/piton1.html [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+				<li><a href="./piton2.html" target="_blank"class="bluetext">Программа на Python — часть 2</a></li>
+				<li><a href="./piton3.html" target="_blank"class="bluetext">Программа на Python — часть 3</a></li>
+			</ul>	
+		<a name="chapter1"><h2 >Содержание</h2></a>
+			
+		<h3>Программа на Python, Часть 1</h3> 
+			<ul>
+				<li><a href="#chapter2"><u class="bluetext">Наша первая программа</u></a></li>
```
Comment (pluhin) [inline AndreiFamishkin/03.links/piton1.html]: need remove tag u, see description for home tasks

File: AndreiFamishkin/03.links/piton2.html [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+        <style>
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+		<h1>HOW-TO: Программа на Python, Часть 2</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>
```
Comment (pluhin) [inline AndreiFamishkin/03.links/piton2.html:15]: Do not use target blank here

File: AndreiFamishkin/03.links/piton2.html [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+                <li><a href="./piton2.html"target="_blank"class="bluetext">Программа на Python — часть 2</a></li>
+                <li><a href="./piton3.html"target="_blank"class="bluetext">Программа на Python — часть 3</a></li>
+            </ul>	
+            <a name="chapter1"><h2 >Содержание</h2></a>
+		<h3>HOW-TO: Программа на Python, Часть 2</h3> 
+			<ul>
+				<li><a href="#chapter2"><u class="bluetext">Списки</u></a></li>
```
Comment (pluhin) [inline AndreiFamishkin/03.links/piton2.html]: need remove tag u

**Review by pluhin — COMMENTED** [review]
Could you please align markup all code section to  one format

File: AndreiFamishkin/03.links/piton3.html [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+            <style>
+                .bluetext {
+                        color: blue;
+                }
+            </style>
+	</HEAD>
+	<BODY> 
+		<h1>HOW-TO: Программа на Python, Часть 3</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>
```
Comment (pluhin) [inline AndreiFamishkin/03.links/piton3.html:15]: remove  target blank

File: AndreiFamishkin/03.links/piton3.html [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+                <li><a href="./piton2.html"target="_blank"class="bluetext">Программа на Python — часть 2</a></li>
+                <li><a href="./piton3.html"target="_blank"class="bluetext">Программа на Python — часть 3</a></li>
+            
+            </ul>
+		<a name="chapter1"><h2 >Содержание</h2></a>
+		<h3>HOW-TO: Программа на Python, Часть 3</h3> 
+			<ul>
+				<li><a href="#chapter2"><u class="bluetext">Модули</u></a></li>
```
Comment (pluhin) [inline AndreiFamishkin/03.links/piton3.html]: remove tag u

File: AndreiFamishkin/04.MultiMedia/piton1.html [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+			<img align="right" src="Python.png" alt="logotip" title="Python">
+		<h1>HOW-TO: Программа на Python, Часть 1</h1>
+			<ul>
+				<li><a href="./piton1.html" target="_blank"class="bluetext">Программа на Python — часть 1</a></li>
```
Comment (pluhin) [inline AndreiFamishkin/04.MultiMedia/piton1.html:16]: remove  target blank

File: AndreiFamishkin/04.MultiMedia/piton1.html [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+				<li><a href="./piton2.html" target="_blank"class="bluetext">Программа на Python — часть 2</a></li>
+				<li><a href="./piton3.html" target="_blank"class="bluetext">Программа на Python — часть 3</a></li>
+			</ul>	
+		<a name="chapter1"><h2 >Содержание</h2></a>
+			
+		<h3>Программа на Python, Часть 1</h3> 
+			<ul>
+				<li><a href="#chapter2"><u class="bluetext">Наша первая программа</u></a></li>
```
Comment (pluhin) [inline AndreiFamishkin/04.MultiMedia/piton1.html:24]: remove  tag u

File: AndreiFamishkin/04.MultiMedia/piton2.html [hunk 16]
```diff
@@ -0,0 +1,72 @@
...
+            .bluetext {
+                    color: blue;
+            }
+        </style>
+	</HEAD>
+	<BODY> 
+            <img align="right" src="Python.png" alt="logotip" title="Python">
+		<h1>HOW-TO: Программа на Python, Часть 2</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>
```
Comment (pluhin) [inline AndreiFamishkin/04.MultiMedia/piton2.html:16]: remove  target blank

File: AndreiFamishkin/04.MultiMedia/piton2.html [hunk 17]
```diff
@@ -0,0 +1,72 @@
...
+                <li><a href="./piton2.html"target="_blank"class="bluetext">Программа на Python — часть 2</a></li>
+                <li><a href="./piton3.html"target="_blank"class="bluetext">Программа на Python — часть 3</a></li>
+            </ul>	
+            <a name="chapter1"><h2 >Содержание</h2></a>
+		<h3>HOW-TO: Программа на Python, Часть 2</h3> 
+			<ul>
+				<li><a href="#chapter2"><u class="bluetext">Списки</u></a></li>
```
Comment (pluhin) [inline AndreiFamishkin/04.MultiMedia/piton2.html:23]: remove  tag u

File: AndreiFamishkin/04.MultiMedia/piton3.html [hunk 18]
```diff
@@ -0,0 +1,114 @@
...
+                .bluetext {
+                        color: blue;
+                }
+            </style>
+	</HEAD>
+	<BODY> 
+            <img align="right" src="Python.png" alt="logotip" title="Python">
+		<h1>HOW-TO: Программа на Python, Часть 3</h1>
+            <ul>
+                <li><a href="./piton1.html"target="_blank"class="bluetext">Программа на Python — часть 1</a></li>
```
Comment (pluhin) [inline AndreiFamishkin/04.MultiMedia/piton3.html:16]: remove  target blank

File: AndreiFamishkin/04.MultiMedia/piton3.html [hunk 19]
```diff
@@ -0,0 +1,114 @@
...
+                <li><a href="./piton2.html"target="_blank"class="bluetext">Программа на Python — часть 2</a></li>
+                <li><a href="./piton3.html"target="_blank"class="bluetext">Программа на Python — часть 3</a></li>
+            
+            </ul>
+		<a name="chapter1"><h2 >Содержание</h2></a>
+		<h3>HOW-TO: Программа на Python, Часть 3</h3> 
+			<ul>
+				<li><a href="#chapter2"><u class="bluetext">Модули</u></a></li>
```
Comment (pluhin) [inline AndreiFamishkin/04.MultiMedia/piton3.html:24]: remove tag u

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by pluhin** [comment]
Any updates?

**Comment by andrei2701** [comment]
today will be22.01.2019, 21:09, "Siarhei Pishchyk" <notifications@github.com>:Any updates?

**Comment by andrei2701** [comment]
исправлено

**Comment by pluhin** [comment]
> исправлено

Not all, see comments

**Pull request closed without merging**